app/middlewares/user.js

```diff
@@ -34,3 +36,13 @@ exports.checkOrder = [
     .customSanitizer(value => value && value.toUpperCase())
     .isIn([ascOrder, descOrder])
 ];
+
+exports.checkBoughtAlbumsPermission = (req, res, next) => {
+  if (req.userPayload.role === default_role && req.userPayload.id === parseInt(req.params.user_id)) {
+    return next();
+  }
+  if (req.userPayload.role === admin_role) {
```

Comment:
Could be simplified in one condition, separating both cases like `if(isAdmin || ownAlbums)` or something like that